api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -1,353 +1,282 @@
-    option (google.api.http).get = "/cfgmgmt/organizations";
-    // if you can get nodes, you are allowed to get the organization field on nodes
-    option (chef.automate.api.iam.policy).resource = "infra:nodes";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:list";
-  };
-
-  /*
-  GetSourceFqdns
-
-  Returns a list of all Chef Infra Servers associated with nodes that have checked in to Automate.
-
-	Authorization Action:
-	```
-	infra:nodes:list
-	```
-	*/
-  rpc GetSourceFqdns(cfgmgmt.request.SourceFqdns) returns (google.protobuf.ListValue) {
-    option (google.api.http).get = "/cfgmgmt/source_fqdns";
-    // if you can get nodes, you are allowed to get the source fqdn field on nodes
-    option (chef.automate.api.iam.policy).resource = "infra:nodes";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:list";
-  };
-
-  /*
-  GetAttributes
-
-  Returns the latest reported attributes for the provided node ID.
-
-	Authorization Action:
-	```
-	infra:nodes:get
-	```
-	*/
-  rpc GetAttributes (cfgmgmt.request.Node) returns (cfgmgmt.response.NodeAttribute) {
-    option (google.api.http).get = "/cfgmgmt/nodes/{node_id}/attribute";
-    option (chef.automate.api.iam.policy).resource = "infra:nodes:{node_id}";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:get";
-  };
-
-  rpc GetVersion(common.version.VersionInfoRequest) returns (common.version.VersionInfo) {
-    option (google.api.http).get =  "/cfgmgmt/version";
-    option (chef.automate.api.iam.policy).resource = "system:service:version";
-    option (chef.automate.api.iam.policy).action = "system:serviceVersion:get";
-    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-			tags: "hidden";
-		};
-  };
-
-  /*
-  GetPolicyCookbooks
-
-  Returns Policy Names with a list of cookbook names and associated policy identifiers based on a policy revision ID.
-  Policy revision ids are sent with an infra run report and identifies which instance of a policy the node used for this run.
-
-	Authorization Action:
-	```
-	infra:nodes:list
-	```
-	*/
-  rpc GetPolicyCookbooks(cfgmgmt.request.PolicyRevision) returns (cfgmgmt.response.PolicyCookbooks) {
-    option (google.api.http).get =  "/cfgmgmt/policy_revision/{revision_id}";
-    option (chef.automate.api.iam.policy).resource = "infra:nodes:{revision_id}";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:list";
-  };
-
-  /*
-  GetErrors
-
-  Returns a list of the most common errors reported for infra nodes' most recent Chef Infra Client runs.
-
-  Authorization Action:
-  ```
-  infra:nodes:list
-  ```
-  */
-  rpc GetErrors(cfgmgmt.request.Errors) returns (cfgmgmt.response.Errors) {
-    option (google.api.http).get = "/cfgmgmt/errors";
-    option (chef.automate.api.iam.policy).resource = "infra:nodes";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:list";
+  // GetNodes
```

Comment:
⚠️ Our documentation processing scripts appear to depend on the using the `/*` commenting style, so this comment style change is actually a bit of a breaking change.  There is an option in the pretty-printer to prefer the `/*` style, but I fear that might then pull in comments we didn't intend to publish into the docs.